client/src/pages/Favorites.js

```diff
@@ -0,0 +1,64 @@
+import { Grid, Card } from "semantic-ui-react"
+import { useState, useEffect } from "react";
+
+export default function Favorites() {
+  const [favorites, setFavorites] = useState([]);
+
+  useEffect(() => {
+    async function getFavoritesData() {
+      try {
+        const getFavorites = await fetch('/api/public/Tables/Favorites', {
+          method: "GET",
+          headers: {
+            'Content-Type': 'application/json'
+          }
+        });
+        if (!getFavorites.ok) {
+          throw new Error('Failed to load recipes');
+        }
+        const recipeData = await getFavorites.json();
+        setFavorites(recipeData);
+      } catch (err) {
+        console.error(err);
```

Comment:
Look at all the `fetch` calls and figure out how to show an error message to the user.